gitversion/Readme.md

```diff
@@ -0,0 +1,19 @@
+# Pelican gutversion Plugin
```

Comment:
`gitversion` is misspelled here.

Reply:
Oh, especially stupid typo, thanks for the catch. I have added more commits to this PR, i this the correct way or should I create a new one?

Reply:
Oh, there's definitely no reason to create a new pull request for fixing things about the original pull request. The best solution is indeed to add new commits, and once everything has been approved, [rebase & squash your commits](https://github.com/getpelican/pelican/wiki/Git-Tips#squashing-commits).